### ISSUE TITLE
[enhancement] request: pre-select a specific newsletter in "subscribe page"

### ISSUE THREAD

**Issue by talgalili** [phpList/phplist3#806]
When creating a subscribe page with more than one list, the user is asked to mark v on the lists they wish to join.
However, if there are several lists, the user is expected to actively choose.
But sometimes, we know that most users will choose a specific list, so it makes sense to pre-select it (and the user can un-check that list if they want).

Either adding that option in the subscribe page setting would be great, or by adding some javascript that allows to pass a url parameter so to pre-select some specific list (this is a hack, so less preferred, but also might be useful sometimes)

**Comment by michield** [comment]
Yes, that's an interesting idea. I think it'll be best placed on the subscribe page management page. Some additional option to say "check by default" for each list. I'd happily accept a PR with that change.
